### PR TITLE
Feat/enable command args for datamodel se

### PR DESCRIPTION
**Pull request by prmukherj**

### CODE REVIEW

File: tests/test_meshing_workflow.py [hunk 1]
```diff
@@ -180,3 +183,29 @@ def test_meshing_workflow_raises_exception_on_invalid_key_in_task_args_2(
...
+    os.environ["PYFLUENT_FLUENT_ROOT"] = r"C:\ANSYSDev\ANSYSDev\vNNN\fluent"
+    # -----------------------------------------------------------------------
+
+    session_new = pf.launch_fluent(mode="meshing")
+    w = session_new.workflow
+    w.InitializeWorkflow(WorkflowType="Watertight Geometry")
+    igt = w.task("Import Geometry")
+    assert igt.CommandArguments.CadImportOptions()
+    assert igt.CommandArguments.CadImportOptions.OneZonePer()
+    assert igt.CommandArguments.CadImportOptions.OneZonePer.getAttribValue("default")
```
Comment (seanpearsonuk) [inline tests/test_meshing_workflow.py:211]: Right, so this works without additional changes, great.

**Comment by seanpearsonuk** [comment]
@prmukherj I have restarted the CI.  tuigen was failing which shouldn't be triggered by these changes.

File: src/ansys/fluent/core/services/datamodel_tui.py [hunk 2]
```diff
@@ -210,12 +210,12 @@ def get_static_info(self) -> Dict[str, Any]:
         DataModelProtoModule.StaticInfo
             static info
         """
-        if hasattr(DataModelProtoModule, "GetStaticInfoRequest"):
+        try:
             request = DataModelProtoModule.GetStaticInfoRequest()
             request.path = self._path
             response = self._service.get_static_info(request)
```
Comment (mkundu1) [inline src/ansys/fluent/core/services/datamodel_tui.py:216]: This wil throw in Fluent 22.2

**Review by mkundu1 — APPROVED** [review]

**Comment by seanpearsonuk** [comment]
@mkundu1 @prmukherj 

There seem to be two classes of errors in the 23R1 failing tests:

- mysteriously missing TUI menus. Some menus are missing in these tests, but I see them locally in PyFluent and in the Fluent TUI.  Those affect meshing and solver

- settings API updates between 22R2 and 23R1 that are not reflected in these tests:
   (a)  difference between {'constant': 0, 'option': 'constant or expression'} and later {'option': 'value', 'value': 0} 
   (b)  difference between materials.copy_database_material_by_name(...) and later
materials.database.copy_by_name(...)
   for each of these I propose a test util wrapper function: 
   (a) compare_constants and (b) copy_material

Are we able to check the backend version from within the test?

**Comment by seanpearsonuk** [comment]
> @mkundu1 @prmukherj
> 
> There seem to be two classes of errors in the 23R1 failing tests:
> 
> * mysteriously missing TUI menus. Some menus are missing in these tests, but I see them locally in PyFluent and in the Fluent TUI.  Those affect meshing and solver
> * settings API updates between 22R2 and 23R1 that are not reflected in these tests:
>   (a)  difference between {'constant': 0, 'option': 'constant or expression'} and later {'option': 'value', 'value': 0}
>   (b)  difference between materials.copy_database_material_by_name(...) and later materials.database.copy_by_name(...)
>   for each of these I propose a test util wrapper function:
>   (a) compare_constants and (b) copy_material
> 
> Are we able to check the backend version from within the test?

I have merged some test changes so that all settings API issues are resolved. We are now down to just those TUI API issues.
@mkundu1 @prmukherj

**Review by seanpearsonuk — APPROVED** [review]

**Pull request merged**